pkg/core/object/errors.go

```diff
@@ -12,3 +12,8 @@ var ErrRangeOutOfBounds = errors.New("payload range is out of bounds")
 
 // ErrAlreadyRemoved returned when object has tombstone in graveyard.
 var ErrAlreadyRemoved = errors.New("object already removed")
+
+// ErrEndOfListing returns from object listing with cursor when storage
+// can't return any more objects after provided cursor.
+// Use empty cursor to start listing again.
+var ErrEndOfListing = errors.New("end of object listing")
```

Comment:
This is definitely not a status error. If `core/object/errors.go` contains only status errors, then it should be moved.

This error returned by metabase but I would like to hide this detail and define in in other package. Engine package seems okay, but it will look like this:

```go
package engine

var ErrEndOfListing = meta.ErrEndOfListing
```

Reply:
There is no conditionality of placing errors in this package, so for now let it be here.